pandapipes/component_models/component_toolbox.py

```diff
@@ -197,4 +197,24 @@ def standard_branch_wo_internals_result_lookup(net):
     else:
         required_results_hyd.extend([("v_mean_m_per_s", "v_mps")])
 
-    return required_results_hyd, required_results_ht
\ No newline at end of file
+    return required_results_hyd, required_results_ht
+
+
+def get_component_array(net, component_name, internal=True):
```

Comment:
I do not understand, why you call it internal?

Reply:
Thought it was called internal_pit, but maybe we should rename it to only_active.

Reply:
I updated this behavior now.

Reply:
Perfect